packages/openapi-v2/test/unit/controller-spec/controller-decorators.test.ts

```diff
@@ -52,15 +49,11 @@ describe('Routing metadata', () => {
   });
 
   it('returns spec defined via @get decorator', () => {
-    const operationSpec = anOperationSpec()
-      .withStringResponse()
-      .build();
+    const operationSpec = givenAnOperationSpec();
 
     class MyController {
       @get('/greet', operationSpec)
-      greet() {
-        return 'Hello world!';
-      }
+      greet() {}
```

Comment:
@b-admike I am not quite familiar with these test cases, but I feel the spec for a function that has a return arg is different than the one that doesn't have a return...so I am not sure how the same `openrationSpec` passes for both?

Maybe the code with your change is correct. It depends on does `operationSpec` contains a response object.

Reply:
@jannyHou I think these tests are for asserting that the correct operation object is mapped to its corresponding controller method. Yeah I think the response object in `operationSpec` is the same throughout the tests.